addons/trait-pyroscope/README.md

```diff
@@ -0,0 +1,80 @@
+# Supported workload type
+Pyroscope Trait supports following component types: webservice、worker and cloneset.
+
+# How to start
+- Use a component typed webservice to start, keep the following to app-demo.yaml, then vela up -f app-demo.yaml
```

Comment:
why we need these trait-pryoscope files in this addon?

Reply:
I will close this pr, new a pr instead: https://github.com/kubevela/catalog/pull/364#issue-1261527441